apps/query-eval/queryeval/driver.py

```diff
@@ -255,12 +255,30 @@ def do_eval(self, _query: QueryEvalQuery, result: QueryEvalResult) -> QueryEvalR
             console.print("[yellow]:point_right: Dry run: skipping eval")
             return result
 
+        # Evalute query plans
+        if not _query.expected_plan:
+            console.print("[yellow]:construction: No expected query plan found, skipping.. ")
+        elif not result.plan:
+            console.print("[yellow]:construction: No computed query plan found, skipping.. ")
+        else:
+            plan_diff = _query.expected_plan.compare(result.plan)
+            if len(plan_diff) == 0:
+                console.print("[green]\U00002714 Plan correct")
+            else:
+                console.print("[red]:x: Plan incorrect")
```

Comment:
Maybe "Plan mismatch" (and "Plan match") since "correct" is of course dependent on many things...

Reply:
Done